### ISSUE TITLE
Moving files

### ISSUE THREAD

**Issue by IncognitoFawkes** [xournalpp/xournalpp#5508]
### Is Your Feature Request Related to a Problem?

I had this one big folder that I moved from my C: drive to my D: drive. It had pdfs and the xpp files of those pdfs. On moving this folder, the xpp file can no longer find the pdf, and I'm required to reselect the pdf background for each xournal file. Is there a fix for this.

### Feature Description

I am not sure how the issue can be fixed, but is there a fix for the issue mentioned above.

**Comment by IncognitoFawkes** [comment]
Nvm it does find the pdf, it just doesn't choose it automatically.

**Comment by eldipa** [comment]
Yes, `xournal++` has a link to the PDF file and moving it (changing its location) breaks the link. You can relink it (if I don't recall wrong, once you open the app it will tell you that the PDF file is missing and it will offer to search it).

By the moment the `xopp` file format does not support other schema (well, there are some variants, but all has some tradeoffs). In #937 we are designing a new file format with the hope to make this situations easier.
